### ISSUE TITLE
Parse the two hyphens in the HTML comment as one hyphen

### ISSUE THREAD

**Issue by ota-meshi** [mysticatea/vue-eslint-parser#72]
I would like to be able to accept directive comment descriptions in `eslint-plugin-vue`.
However, the comment value did not produce the expected result.

The following HTML comment is parsed as follows:

```vue
<template>
  <!-- eslint-disable -- description -->
</template>
```

```json5
[
  {
    "type": "HTMLComment",
    "range": [ /* ...  */ ],
    "loc": { /* ...  */ },
    "value": " eslint-disable - description "
  }
]
```

I expect the following results.

```diff
[
  {
    "type": "HTMLComment",
    "range": [ /* ...  */ ],
    "loc": { /* ...  */ },
-    "value": " eslint-disable - description "
+    "value": " eslint-disable -- description "
  }
]
```

---

I've heard that the old HTML specification should avoid consecutive hyphens.
Do you think you should avoid separating the description with two hyphens?

**Comment by mysticatea** [comment]
Thank you for your report.

It looks like a bug.

The HTML comment parsing is defined in https://html.spec.whatwg.org/multipage/parsing.html, from the section 12.2.5.43 to 12.2.5.52. It's complex a bit.

"Comment state" -> "Comment end dash state" -> "Comment end state", then it says "Append *two* U+002D HYPHEN-MINUS characters (-) to the comment token's data.", but [the tokenizer appends only one hypen](https://github.com/mysticatea/vue-eslint-parser/blob/master/src/html/tokenizer.ts#L1493).

**Comment by ota-meshi** [comment]
Thank you for your comment!
I have opened a PR to fix this.
